src/components/resource/user/NoteUpdate.vue

```diff
@@ -36,8 +36,7 @@ export default {
 
       this.$store.dispatch('user/update', { params: payload, userId: this.userId })
         .then(() => {
-          this.note = this.noteField
-          this.reset()
+          this.dialog = false
```

Comment:
C'est intéressant comme bug fix en review, car je pensais que c'était correct et j'aurai rien vu.

Et en vrai maintenant, je ne vois pas comment est mis à jour dans la vue la note affichée en mode 'normal' après le dispatch.
en gros où est fait le `this.note = this.noteField` (ou action similaire) ?

Reply:
Je pense que le bug vient du fait que la boite de dialog est jamais détruite tant que tu restes sur la page.

Je fais un `get` /users/:uid/show après l'update, il met à jour le state dans le store et la computed property se met à jour toute seule (vue's magic !)

Reply:
oui en effet je n'avais pas fais attention au `.then(() => dispatch('get'))` dans le store...

c'est clairement ça, vu que c'est à jour dans le back tu fais un refresh de l'API qui te donne l'info à jour et vue-js fais le reste

Reply:
La v2 ce sera d'utiliser la réponse du serveur lors de l'update pour mettre le store à jour plutôt que de refaire un get, mais on a pas encore de soucis d'optimisation à ce point.